cpp/modmesh/python/module.cpp

```diff
@@ -52,6 +53,7 @@ void initialize(pybind11::module_ mod)
 #else // QT_CORE_LIB
     mod.attr("HAS_VIEW") = false;
 #endif // QT_CORE_LIB
+    wrap_Config(mod);
```

Comment:
I suppose this line will make `modmesh.config` work, but I don't know why it doesn't work.